src/Database/Dialect/PostgresDialectTrait.php

```diff
@@ -129,6 +129,24 @@ protected function _transformFunctionExpression(FunctionExpression $expression)
             case 'NOW':
                 $expression->name('LOCALTIMESTAMP')->add([' 0 ' => 'literal']);
                 break;
+            case 'DATE_ADD':
+                $expression
+                    ->name('')
+                    ->type(' + INTERVAL')
+                    ->iterateParts(function ($p, $key) {
+                        if ($key === 1) {
+                            $p = sprintf("'%s'", $p);
```

Comment:
It is combination of    number and unit e.g.  `'1 hour'`   in this:   `created + INTERVAL '1 hour'`
Ref: http://www.postgresql.org/docs/9.0/static/functions-datetime.html